docs/Formulas.md

```diff
@@ -0,0 +1,69 @@
+# How pe_tune calculates values
+
+## Memory
+
+### Minimums and Maximums
+in MBs
+puppetserver min = 512  
+code_cache min   = 128  
+code_cache max   = 2048  
+database min     = 2048  
+database max     = 16384  
+puppetdb min     = 512  
+puppetdb max     = 8192  
+orchestrator min = 512  
+orchestrator max = 1024  
+console min      = 512  
+console max      = 1024  
+activemq min     = 512  
+activemq max     = 1024  
+
+###Standard ref arch (single host)
+
+master Memory = 
+1GB os + 25% database + 10% puppetdb + 8% orch + 8% console + 8% mco(if present) 
+
+lovejoy and on, orchestrator gets extra ram equal to one puppetserver jruby (see below)
```

Comment:
orchestrator receives additional memory, equal to what would be assigned to one puppetserver jruby
puppetserver receives one less processor, therefore one less jruby